dev/docker/hive/hive-dependency.sh

```diff
@@ -89,7 +90,9 @@ if [ ! -f "${hive_dir}/packages/${ZOOKEEPER_PACKAGE_NAME}" ]; then
 fi
 
 if [ ! -f "${hive_dir}/packages/${RANGER_HDFS_PACKAGE_NAME}" ]; then
-  bash ${hive_dir}/download-release.sh "v${RANGER_VERSION}" ${RANGER_HDFS_PACKAGE_NAME} "${hive_dir}/packages/${RANGER_HDFS_PACKAGE_NAME}"
```

Comment:
@unknowntpo I think we can delete `dev/docker/hive/download-release.sh` in this PR. because we didn't neet it anymore.

Reply:
Ok, removed.